Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I don't know that adding a comment at the top of every file is preferable to just freezing the few string constants that we have.